src/components/MessagesList/MessagesList.vue

```diff
@@ -1075,7 +1075,7 @@ export default {
 
 .scroll-to-bottom {
 	position: absolute !important;
-	bottom: 76px;
+	bottom: 140px;
```

Comment:
To make this property reliable on typing indicator, we need to pass it somehow through the parent component (which I think is an overkill)